components/micro-gateway-core/src/main/ballerina/gateway/filters/subscription_filter.bal

```diff
@@ -82,6 +83,25 @@ public type SubscriptionFilter object {
                                     return true;
                                 }
                             }
+                            }
+                            else
+                            {
+                                authenticationContext.authenticated = true;
+                                authenticationContext.tier = "Unlimited";
+                                authenticationContext.apiKey = jwtToken;
+                                authenticationContext.applicationId = "";
+                                authenticationContext.applicationName = "";
+                                authenticationContext.applicationTier = "Unlimited";
```

Comment:
Use a constant for Unlimited

Reply:
this code has revamped and formatted with 4d34de4